src/groups/mqb/mqbblp/mqbblp_localqueue.cpp

```diff
@@ -538,7 +533,7 @@ void LocalQueue::postMessage(const bmqp::PutHeader&              putHeader,
 
     // If 'FileStore::d_storageEventBuilder' is flushed, flush all relevant
     // queues (call 'afterNewMessage' to deliver accumulated data)
-    d_state_p->storage()->dispatcherFlush(false, true);
+    d_state_p->storage()->flushQueues();
```

Comment:
This (already existing) logic is a bit unclear.  Please, check the following reasoning:
1.  `LocalQueue::postMessage` calls `put` which adds to the builder in the `FileStore`.
2.  `FileStore` tracks the builder size and  does `FileStore::flushIfNeeded` which used to call `notifyQueuesOnReplicatedBatch`
3.  Then, why do we need this call `dispatcherFlush(false, true)`/`flushQueues()` here?

If we don't, then there is no need to have `flushQueues` as an interface method.  It is entirely private business of `FileStore` (and something like `notifyQueuesOnReplicatedBatch` seems like a better name)

Reply:
That is correct, this call is unnecessary. Removed & simplified